Sources/SwiftRouting/Router/Router.swift

```diff
@@ -87,16 +89,27 @@ extension Router: @preconcurrency RouterModel {
     log(.action, message: "popToRoot")
   }
 
-  @MainActor public func close() {
-    if type.isPresented {
-      triggerClose = true
-      log(.action, message: "close")
+  @MainActor public func close<T: TerminationRoute>(_ value: T? = nil) {
```

Comment:
RouteTermination might be more accurate I believe? But I'm not native English :)